releasenotes/notes/cmd2-0.9.0-35b6c209cbea798f.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
No need for this reno here given that this is the next RC. We only need this in the patch to the RC branch as that will become the next release.